obelix/idefix/trigger/dispatcher.go

```diff
@@ -0,0 +1,16 @@
+package trigger
+
+import (
+	"github.com/Shopify/sarama"
+	"idefix/operator"
+	"idefix/redis"
+	"reflect"
+)
+
+type Dispatcher interface {
+	Parse(data []byte) (reflect.Value, error)
+	LookForChange(op *operator.IdefixOperator, key, old string) error
+	SendToKafka(kp sarama.SyncProducer, workflowID string) error
+	GetRedisState(rc *redis.Client, key string) (string, error)
+	UpdateRedisState(rc *redis.Client, key, newer string) error
```

Comment:
UpdateRedisState is never called in public, should you really keep it?

Reply:
implem interface

Reply:
Yes but why implement a method that will be never use

Reply:
f8f0aaf50c0b54cf567af321f6f09f4d670af7e2